superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -1369,19 +1368,15 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
                     )}
                   </StyledRadioGroup>
                 </div>
-                {isReport && (
-                  <div className="inline-container">
-                    <StyledCheckbox
-                      className="checkbox"
-                      checked={forceScreenshot}
-                      onChange={onForceScreenshotChange}
-                    >
-                      Ignore cache when generating screenshot
-                    </StyledCheckbox>
-                  </div>
-                )}
               </>
             )}
+            <StyledCheckbox
```

Comment:
should this have a tooltip that explains what it is? Or is it self-explanatory

Reply:
it's not in the spec, personally think it is fine since we are explicitly putting text right next to the checkbox